### DISCUSSION TITLE
Surgar-Cane should be growable via Bonemeal!

### DISCUSSION THREAD

**Discussion by Tydium** [PurpurMC/Purpur#1124] [__Dismissed__] [answered]
This is currently a bedrock native feature, and when I found out it wasn't on Java I was quite surprised. I guess you can say this is a self explanatory feature also. Just a simple Bone-meal on sugar-cane and it grows taller. Just like Bedrock Edition behavior!

**Comment by Keeg-Turner** [comment, marked as answer]
Duplicate of https://github.com/PurpurMC/Purpur/discussions/1096
